packages/components/datepicker/stories/Calendar.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+
+import { Calendar } from '../src';
+import type { CalendarProps } from '../src';
+import { DateRange } from 'react-day-picker';
+
+export default {
+  component: Calendar,
+  title: 'Components/Datepicker/Calendar',
+} as Meta;
+
+export const Default: Story<CalendarProps> = (args) => {
```

Comment:
I think the website page will also need some love and good examples

Reply:
<img width="332" alt="Screenshot 2022-06-02 at 16 58 14" src="https://user-images.githubusercontent.com/6163988/171664584-8251b342-2be9-41e4-93d3-ac291c54ff2d.png">
Menu structure should be under date components section

Reply:
will be done in the separate documentation story